sdk-api-src/content/winusb/nf-winusb-winusb_readisochpipeasap.md

```diff
@@ -108,13 +108,14 @@ If the caller sets <i>ContinueStream</i> to TRUE, The transfer fails if Winusb.s
 
 
 
-<b>WinUsb_ReadIsochPipeAsap</b> allows the USB driver stack to choose the starting frame number for the transfer.  If one or more transfers are already pending on the endpoint, the transfer will be scheduled for the frame number immediately following the last frame number of the last currently pending transfer.
+<b>WinUsb_ReadIsochPipeAsap</b> allows the USB driver stack to choose the starting frame number for the transfer. If one or more transfers are already pending on the endpoint, the transfer will be scheduled for the frame number immediately following the last frame number of the last currently pending transfer.
 
 
-<b>WinUsb_ReadIsochPipeAsap</b> packetizes the transfer buffer so that in each interval,  the host can send the maximum bytes allowed per interval. The maximum bytes is as specified by the endpoint descriptor for full and high-speed endpoints, and endpoint companion descriptor for SuperSpeed endpoints.
+<b>WinUsb_ReadIsochPipeAsap</b> packetizes the transfer buffer so that in each 1ms interval,  the host can send the maximum bytes allowed per interval. The maximum bytes is as specified by the endpoint descriptor for full and high-speed endpoints, and endpoint companion descriptor for SuperSpeed endpoints.
```

Comment:
1) Packetization is at the micro-frame (125us) level for high-speed/SuperSpeed endpoints, so saying "1ms interval" is not really accurate.
2) Should say the host can **receive** the maximum bytes allowed.

Reply:
1. Agree that this is not accurate. Intention is to document the consequence of packetization for low-latency applications. This is done by the insertion at line 113. Remove this change.
2. True. Fixed in PR https://github.com/MicrosoftDocs/sdk-api/pull/33

Reply:
```suggestion
<b>WinUsb_ReadIsochPipeAsap</b> packetizes the transfer buffer so that in each interval, the host can send the maximum bytes allowed per interval. The maximum bytes is as specified by the endpoint descriptor for full and high-speed endpoints, and endpoint companion descriptor for SuperSpeed endpoints.
```